content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -266,9 +266,22 @@ This ensures that DaemonSet pods are never evicted due to these problems.
 
 ## Taint Nodes by Condition
 
-The node lifecycle controller automatically creates taints corresponding to
-Node conditions with `NoSchedule` effect.
-Similarly the scheduler does not check Node conditions; instead the scheduler checks taints. This assures that Node conditions don't affect what's scheduled onto the Node. The user can choose to ignore some of the Node's problems (represented as Node conditions) by adding appropriate Pod tolerations.
+The node controller automatically creates taints with a `NoSchedule` effect for
+[node conditions](/docs/concepts/scheduling-eviction/pod-eviction#node-conditions).
+The scheduler checks taints, not node conditions, when it makes scheduling
+decisions. This ensures that node conditions don't directly affect scheduling.
+You can ignore node conditions for newly created pods by adding the corresponding
```

Comment:
Move this sentence after the next paragraph. Then, the expression "For example" below is more clearly referring to the taints, and not how users can ignore them.

Reply:
This was not resolved yet.

Reply:
Done. PTAL. Thanks for the reminder!